src/App.tsx

```diff
@@ -1,15 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+// import { event } from 'cypress/types/jquery';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(({ title, description }) => {
+    return title.toLowerCase().includes(query.toLowerCase())
+    || description.toLowerCase().includes(query.toLowerCase());
```

Comment:
First of all, you have forgotten to check if the user puts spaces in the input
Second, maybe I think it's better to put the check into a separate function and reuse it when needed